google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -502,7 +502,7 @@ private void maybeWaitForInflightQuota() {
 
   @VisibleForTesting
   static long calculateSleepTimeMilli(long retryCount) {
-    return Math.min((long) Math.pow(2, retryCount), 60000);
+    return Math.min((long) Math.pow(2, retryCount) * 1000, 60000);
```

Comment:
Thanks for submitting this, the original intention for retrying reconnection at so frequent level is the majority of the connection reconnection does not need seconds of waiting. Were you seeing retry failure with small retry interval?

Reply:
The problem I saw is probably more throttling rather than specific failures, this is used directly in the bigquery-spark-connector and the problem I am seeing is the connector slowing down to a failure writing data into bigquery, this is one of the two points I found led to this (there might be more, I might be mistaken too), Seconds maybe too much yes, bit retrying within a few millis is too aggressive I think, how about we do a base of 50ms or 100ms instead of 1s? That would get the best of both worlds, retry fast the first few times, and then back off exponentially

Reply:
Yeah, I think 50ms makes sense to me. Let's use that value

Reply:
Updated to 50ms